easybuild/easyconfigs/d/DFT-D3/DFT-D3-3.1.1-intel-2015a.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'MakeCp'
+
+name = 'DFT-D3'
+version = '3.1.1'
+
+homepage = 'http://www.thch.uni-bonn.de/tc/index.php?section=downloads&subsection=DFT-D3&lang=english'
+description = """DFT-D3 implements a dispersion correction for density functionals, Hartree-Fock and semi-empirical quantum chemical methods."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = ['http://www.thch.uni-bonn.de/tc/downloads/%(name)s/data']
+# Note that the DFT-D3 tarball is named as "dftd3.tgz" with no version
+# numbering. Also, the authors are prone (alas) to stealth upgrades, so that two
+# tarballs with the same version number can have different checksums. For this
+# reason, it is suggested to manually download and rename the tarball. The
+# checksum may also need updating from time to time.
+sources = ['dftd3-%(version)s.tgz']
+checksums = [('md5', '88470bd398b5cc90ad680ed1df4239f7')]
```

Comment:
hmm, I'm getting `5d1cbaa851cd88ae515dea3cc9a7590d`

and it still seems to be v3.1.1?